app/helpers/application_helper.rb

```diff
@@ -119,53 +135,14 @@ def how_it_works_tiles
       }
     ]
   end
-
-  def courses
```

Comment:
this method is needed to render the badges for courses, can you put it back in please? 😄